XRTK-Core/Packages/com.xrtk.core/Definitions/Platforms/AndroidPlatform.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace XRTK.Definitions.Platforms
+{
+    /// <summary>
+    /// Used by the XRTK to signal that the feature is available on the Android platform.
+    /// </summary>
+    public class AndroidPlatform : BasePlatform
```

Comment:
I see `AndroidPlatform` is defined in Core but `LuminPlatform` in the Lumin module. Wouldn't it make sense to define all the possible XRTK platforms in Core?

Reply:
That's actually the benefit of this change so we don't have to define everything in the core